src/Bridge/Doctrine/Orm/Extension/EagerLoadingExtension.php

```diff
@@ -35,6 +36,8 @@
  */
 final class EagerLoadingExtension implements QueryCollectionExtensionInterface, QueryItemExtensionInterface
 {
+    use ShouldEagerLoad;
```

Comment:
I thought we don't like traits anymore lol... And the naming is just weird. :stuck_out_tongue: